openassessment/workflow/models.py

```diff
@@ -180,21 +180,6 @@ def score(self):
         """
         return sub_api.get_latest_score_for_submission(self.submission_uuid)
 
-    @property
-    def override_score(self):
-        """
-        Latest override score.
-        
-        Note an override score has no submission associated with it.
```

Comment:
Good point about the existing overrides, I'll do some testing on my devstack for that